recipes/sqlite3/all/CMakeLists.txt

```diff
@@ -127,6 +131,12 @@ if(THREADSAFE)
     target_link_libraries(${PROJECT_NAME}  PRIVATE Threads::Threads)
 endif()
 
+if(ENABLE_ICU)
+    find_package(ICU REQUIRED COMPONENTS data i18n uc)
```

Comment:
`cmake_find_package` generator should be added to `conanfile.py`

Reply:
I'm sorry, I don't get how I should change that part of the code. Should I add something to the `_module_file_rel_path` property or add it somewhere else?

Reply:
No, it's unrelated.
Just replace `generators = "cmake"` by `generators = "cmake", "cmake_find_package"`